pyiron_base/job/external.py

```diff
@@ -36,8 +36,8 @@ def get_custom_dict():
         hdf_file = project_folder / folder
         hdf_file = str(hdf_file) + ".h5"
         if Path(hdf_file).exists():
-            hdf = FileHDFio(hdf_file)
-            custom_dict = hdf[folder + '/input/custom_dict/data']
+            hdf = FileHDFio(hdf_file)._create_project_from_hdf5()
```

Comment:
I guess using an internal function is not recommended. Maybe we should make it a public function.

Reply:
The same method is also used by `GenericJob`, so making it public is probably ok.

Reply:
See #268.